lib/src/http/content_type.rs

```diff
@@ -104,6 +104,7 @@ impl ContentType {
         "HTML", HTML, is_html => "text", "html" ; "charset=utf-8",
         "Plain", Plain, is_plain => "text", "plain" ; "charset=utf-8",
         "JSON", JSON, is_json => "application", "json",
+        "MsgPack", MsgPack, is_msgpack => "application", "msgpack",
```

Comment:
`application/msgpack` seems like what we should use here. But see my inline comments about what we should accept.